File: scripts/py-compile-suite/inputs/27_multiple_assignment.m
% Multiple return value unpacking
A = magic(5);
[m, n] = size(A);
[max_val, max_idx] = max(A(:));
[min_val, min_idx] = min(A(:));
[U, S, V] = svd(A);
[~, peak_idx] = max(A(1, :));

File: scripts/py-compile-suite/outputs/27_multiple_assignment.py
import numpy as np

# Multiple return value unpacking
A = magic(5)
m, n = np.atleast_2d(A).shape
A = np.atleast_2d(A)
max_val, max_idx = np.amax(A.flatten(order="F")), np.argmax(A.flatten(order="F"))
min_val, min_idx = np.amin(A.flatten(order="F")), np.argmin(A.flatten(order="F"))
U, S, V = np.linalg.svd(A)
peak_idx = np.argmax(A[0:0+1, :])
